ci/owncloud/owncloud_definitions.sh

```diff
@@ -117,6 +113,8 @@ OWNCLOUD_PUBLIC_URLS["artifacts"]="https://ftp.essensium.com/owncloud/index.php/
 
 user=$(awk '/ftp.essensium.com/{getline; print $4}' ~/.netrc)
 
-if [ ! "$#" = 0 ] ; then
-    "$@"
-fi
+[[ "$_" != "$0" ]] || {
+    if [ ! "$#" = 0 ] ; then
```

Comment:
Actually I tested it again and apparently we don't need the extra `[[ "$_" != "$0" ]]`, so removing.

Reply:
So apparently this is a buggy way to do this, since it doesn't work when sourcing the script from a different script, I changed to use ${BASH_SOURCE[0]} = $0 condition to check if the script is being sourced.